tests/fsharp/core/signedtests/build.bat

```diff
@@ -0,0 +1,265 @@
+@if "%_echo%"=="" echo off
```

Comment:
@enricosada I think it's very important that you submit a PR to remove all the .bat files if they are no longer being used.  It seems they are still in the tree?